pkg/providers/vsphere/upgrade/virtualmachine/vm_schema_upgrade.go

```diff
@@ -54,43 +59,117 @@ func ReconcileSchemaUpgrade(
 		panic("moVM.config is nil")
 	}
 
-	logger := pkglog.FromContextOrDefault(ctx)
+	var (
+		logger   = pkglog.FromContextOrDefault(ctx)
+		features = pkgcfg.FromContext(ctx).Features
+	)
+
 	logger.V(4).Info("Reconciling schema upgrade for VM")
 
+	if vmopv1util.IsVirtualMachineSchemaUpgraded(ctx, *vm) {
+
+		if features.AllDisksArePVCs || features.VMSharedDisks {
+			// Ensure the backfill condition is marked ready since the backfill
+			// reconcile will not be called again since it's been marked as
+			// upgraded.
+			pkgcond.MarkTrue(vm, vmconfunmanagedvolsfill.Condition)
```

Comment:
nit: 

Should this condition be renamed from `VirtualMachineUnmanagedVolumesBackfill` to `VirtualMachineUnmanagedVolumesBackfilled`?

Reply:
I mean, maybe? Not worth updating PR IMO. If I see other changes I need to make, I'll consider it.